config/initializers/vault.rb

```diff
@@ -1,4 +1,9 @@
-VAULT_ENABLED = ENV.fetch('SECRET_STORAGE_BACKEND', false)
+# some hackery to get this loaded up when testing
+VAULT_ENABLED = ENV.fetch('SECRET_STORAGE_BACKEND', Rails.env.test? ? "SecretStorage::HashicorpVault" : false)
```

Comment:
also vault cannot be a test dependency ... most users and travis won't have it ...